dsl_parser/tests/test_functions.py

```diff
@@ -1366,107 +1404,112 @@ def test_get_property_doesnt_accept_runtime_func_as_args(self):
         properties:
             dummy_prop: dummy_value
             prop:
-                get_property:
-                    - concat:
-                        - get_attribute:
-                                - some_node
-                                - dummy_prop
-                    - shouldn't matter
+                get_input:
+                    - get_attribute:
+                            - some_node
+                            - dummy_prop
     """
+        plan = self.parse(yaml)
         with self.assertRaisesRegexp(
-                exceptions.FunctionValidationError,
-                'Runtime function .+ cannot be nested within a non-runtime '
-                'function \\(found in .+\\)'):
-            prepare_deployment_plan(self.parse_1_1(yaml))
+                exceptions.FunctionEvaluationError, 'unresolved argument'):
+            prepare_deployment_plan(plan)
 
-    def test_circular_with_dict_argument(self):
         yaml = """
 node_types:
     some_type:
         properties:
             prop: {}
+            dummy_prop: {}
 node_templates:
     some_node:
         type: some_type
         properties:
+            dummy_prop: dummy_value
             prop:
                 get_input:
-                    get_property:
-                        - some_node
-                        - prop
+                    concat:
+                        - get_attribute:
+                                - some_node
+                                - dummy_prop
     """
+        plan = self.parse_1_1(yaml)
```

Comment:
why use parse_1_1?

Reply:
that's what the original test used, but i now made sure that it's easier to see in the commit diff